SPECS/python3/python3.spec

```diff
@@ -134,10 +134,17 @@ The test package contains all regression tests for Python as well as the modules
 %autosetup -p1 -n Python-%{version}
 
 %build
-export OPT="%{optflags} %{openssl_flags}"
+# Remove GCC specs and build environment linker scripts
+# from the flags used when compiling outside of an RPM environment
+# https://fedoraproject.org/wiki/Changes/Python_Extension_Flags
+export CFLAGS="%{extension_cflags} %{openssl_flags}"
+export CFLAGS_NODIST="%{build_cflags} %{openssl_flags}"
```

Comment:
For my own education: why the `*_NODIST` variables use the `build_*` macros?

Reply:
We want to preserve the linker scripts and the build hardening GCC specs when building the actual `python3` package, but we don't want `python3` to store those flags as part of its `sysconfig` module. So, the [`X` and `X_NODIST` flags](https://docs.python.org/3/using/configure.html#envvar-CFLAGS_NODIST) are used to differentiate the two sets of flags (those stored and those used, respectively).

Reply:
Thanks! A new thing learned.:)